internal/resources/ekscluster/resource_ekscluster_test.go

```diff
@@ -43,6 +43,10 @@ func setupHTTPMocks(t *testing.T, clusterName string) {
 	config := testhelper.TestGetDefaultEksAcceptanceConfig()
 	endpoint := os.Getenv("TMC_ENDPOINT")
 
+	if endpoint == "" {
```

Comment:
Let's set the require env variables in the mock test rather than skipping it. Refers gitops resource, the similar change had to be done here

Reply:
Done